NPCs.bb

```diff
@@ -3000,16 +3001,18 @@ Function UpdateNPCs()
 							n\Idle = False
 							n\State = Rand(20, 60)
 							
-							If Rand(300)=1 Then PlaySound2(RustleSFX(Rand(0,2)),Camera, n\obj, 8, Rnd(0.0,0.2))
+							If Rand(300)=1 Then PlaySound2(RustleSFX(Rand(0,2)),Camera, n\Collider, 8, Rnd(0.0,0.2))
 						End If
-					Else
+					EndIf
+					
+					If (Not n\Idle) Then
```

Comment:
If the `If n\Idle Then` block runs and decides to summon 372, 372's model will be shown and `n\Idle` will be set to `False`, but the code that actually positions the model near the player is in the `Else` block. This meant there would be one frame where the model was shown but not positioned yet. I fixed this by replacing `Else` with what you see here so that the model positioning code can run on the same frame the model is shown. It could've also been fixed by moving `ShowEntity` to the `Else` block but that would cause `ShowEntity` to run every frame that 372 is active.